lms/djangoapps/teams/tests/test_views.py

```diff
@@ -597,10 +597,18 @@ def test_without_course_id(self):
     @ddt.data(
         (None, 200, ['Coal Power', 'Nuclear Power', u'sólar power', 'Wind Power'], 'name'),
         ('name', 200, ['Coal Power', 'Nuclear Power', u'sólar power', 'Wind Power'], 'name'),
+        ('team_count', 200, ['Nuclear Power', u'sólar power', 'Coal Power', 'Wind Power'], 'team_count'),
```

Comment:
It may be worth adding a comment to that effect in the code for future maintainers of this file